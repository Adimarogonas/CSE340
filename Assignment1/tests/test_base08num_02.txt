1Ax0
0x08
