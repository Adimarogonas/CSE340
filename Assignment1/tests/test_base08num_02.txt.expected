{1 , NUM , 1}
{Ax0 , ID , 1}
{0x08 , BASE08NUM , 2}
{ , END_OF_FILE , 4}
